### DISCUSSION TITLE
特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by hjm-chan** [Yowkees/keyball#442] [Q&A] [answered]
お世話になります。
keyball61を組み立てていて添付のキーのみ反応しない状態です。
Bキーが効かない状態になっております。
キーソケットのハンダ不良を疑い、ハンダ付け直しをしてみましたが解消しておりません。
対応するダイオードやチェック先を教えていただけないでしょうか？
よろしくお願いいたします。
![image](https://github.com/Yowkees/keyball/assets/93187948/4a379dc8-1bf0-48b2-8cf6-d9212dd01600)

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

反応しないキーに対応するダイオードは写真の黄色の丸のダイオードになります。
キーソケットのはんだ付けと併せてはんだ付けのご確認をお願いいたします。
![289334401-4a379dc8-1bf0-48b2-8cf6-d9212dd01600](https://github.com/Yowkees/keyball/assets/143698349/d13466dd-8119-496a-ad41-9d43c9dcaff7)

**Reply by hjm-chan** [reply]
ありがとうございます！このキーの反応を直すことができました。
追加でのご質問で恐縮ですが、ピンクで囲ったキーが時々まとめて反応しなくなります。
promicro含めて寄せて撮った写真を共有させていただきます。半田付けの怪しい箇所などをご教授いただけないでしょうか？
![image](https://github.com/Yowkees/keyball/assets/93187948/f4971e05-a810-4f97-aa49-4c21fc5faf64)

![image](https://github.com/Yowkees/keyball/assets/93187948/4302b586-e3f4-4254-a8ec-d0d4f76a9ebf)

![image](https://github.com/Yowkees/keyball/assets/93187948/0adf22f2-6f25-41a1-8749-0ab54ad45030)

![image](https://github.com/Yowkees/keyball/assets/93187948/0ce82dc1-7f4b-4759-8cf7-d8a75d33a4e5)

**Comment by Ryo624** [comment]
右手側のキーの不具合を解決していただきましてありがとうございます。

左手側は横一列のキーの反応が無いということで、ProMicro周辺のはんだ付けに原因がある可能性がありますので、
はんだの形が山型に端子と馴染むようなはんだを意識し、隣の端子とショートしないようにはんだ付けのやり直しを実施していただけますと幸いです。

**Comment by Ryo624** [comment]
遅れて重ねてのご連絡となり、大変申し訳ありません。

ProMicroのGNDと書かれた箇所の端子部分がはんだ屑が隣の端子に触れてショートしてしまっているように見えます、こちら端子が隣同士触れないようにはんだ付けをお願いいたします。
